rpc_to_mongo_db.py

```diff
@@ -0,0 +1,258 @@
+                cfg["attributes"]["proxy_identifier"]: cfg["proxy_identifier_value"],
+            }
+        },
+        auth=(rpc_username, rpc_password),
+    )
+    if resp.status_code != 200:
+        print("facility invalid response")
+        return "INVALID RESPONSE"
+    facilities_list = resp.json()
+    present_client_ids = []
+    collection = get_collection(cfg)
+    for facility in facilities_list:
+        resp = requests.post(
+            "{}json/attributesManager/getAttributes".format(cfg["rpc"]["perun_url"]),
+            json={"facility": facility["id"], "attrNames": get_attr_perun_names(cfg)},
+            auth=(rpc_username, rpc_password),
+        )
+        if resp.status_code != 200:
+            print("attributes invalid response")
+            return "INVALID RESPONSE"
+        facility_attrs = resp.json()
+        attrs_dict = get_attr_dict(cfg, facility_attrs)
+        print("Processing client_id: {}.".format(attrs_dict["client_id"]))
+        try:
+            result = collection.replace_one(
+                {"client_id": attrs_dict["client_id"]}, attrs_dict, True
+            )
+            if result.upserted_id:
+                print("Client {} upserted.".format(attrs_dict["client_id"]))
+            else:
+                print("Client {} updated.".format(attrs_dict["client_id"]))
+
+        except OperationFailure as e:
+            print(
+                "Processing of client_id failure code: {} details: {}".format(
+                    attrs_dict["client_id"], e.code, e.details
+                )
+            )
+        present_client_ids.append(attrs_dict["client_id"])
+    if cfg["delete_not_present_clients"]:
+        try:
+            print("Processing delete of not present clients.")
+            result = collection.delete_many({"client_id": {"$nin": present_client_ids}})
+            print("Deleted {} not present clients.".format(result.deleted_count))
+        except OperationFailure as e:
+            print(
+                "Processing of delete not present clients failure code: {} details: {}".format(
+                    e.code, e.details
+                )
+            )
+
+
+def get_issue_refresh_tokens_value(cfg, facility_attrs):
+    for attr in facility_attrs:
+        attr_perun_name = attr["namespace"] + ":" + attr["baseFriendlyName"]
+        if attr_perun_name == cfg["attributes"]["issue_refresh_tokens"]:
+            if attr["value"] == "true":
+                return True
+            return False
+
+
+def get_attr_dict(cfg, facility_attrs):
+    result = {}
+    for key, value in cfg["static_attributes"].items():
+        result[key] = value
+
+    for attr in facility_attrs:
+        attr_perun_name = attr["namespace"] + ":" + attr["baseFriendlyName"]
+        for key, value in cfg["attributes"].items():
+            if isinstance(value, list) and attr_perun_name in value:
+                if key in result.keys():
```

Comment:
```suggestion
                if key in result:
```
With dictionaries, `x in y` does the same thing as `x in y.keys()`